fiftyone/utils/data/exporters.py

```
@@ -2504,6 +2625,65 @@ def close(self, *args):
+            -   ``"symlink"``: create symlinks to each media file in the export
+                directory
+        rel_dir (None): an optional relative directory to strip from each input
+            filepath to generate a unique identifier for each audio. When
+            exporting media, this identifier is joined with ``export_dir`` to
+            generate an output path for each exported audio. This argument
+            allows for populating nested subdirectories that match the shape of
+            the input paths. The path is converted to an absolute path (if
+            necessary) via :func:`fiftyone.core.storage.normalize_path`
+    """
+
+    def __init__(self, export_dir, export_media=None, rel_dir=None):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)